activejob/test/cases/argument_serialization_test.rb

```diff
@@ -154,6 +156,18 @@ class ClassArgument; end
     end
   end
 
+  test "should maintain a functional duration" do
+    duration = perform_round_trip([1.year]).first
+    assert_equal 2.years, duration + 1.year
+  end
+
+  # TODO: remove in 7.1
+  test "should be able to deserialize a duration serialized under the old scheme" do
+    old_serialization = [{ "_aj_serialized" => "ActiveJob::Serializers::DurationSerializer", "value" => 31556952, "parts" => [[{ "_aj_serialized" => "ActiveJob::Serializers::SymbolSerializer", "value" => "years" }, 1]] }]
```

Comment:
💅 We probably want to break up this into multiple lines so it's easier to read (and review).